examples/README.md

```diff
@@ -5,6 +5,8 @@
 ```
 $ ./gradlew installDist
```

Comment:
IIUC, this command now needs to be run inside the `examples` directory, so the next two commands need to be updated.

Reply:
Thanks for pointing out. I'll update that later.